pages/common/irssi.md

```diff
@@ -4,11 +4,11 @@
 
 - Open irssi and connect to a server with a nickname:
 
-`irssi -n {{nickname}} -c {{server.name.com}}`
+`irssi -n {{nickname}} -c {{server.example.com}}`
```

Comment:
Is there any particular reason to use a subdomain in this example?

Reply:
I don't know the command but I believe the subdomain is needed because this is a IRC client. IRC servers will usually run under something like "irc.example.com".

Reply:
Not that it is a must but it makes sense to have that subdomain there.

Reply:
Yeah, I'd vote to keep the subdomain here.

Reply:
I see, thanks for the clarifications. That said, what do you guys think about using `irc.example.com`, or `irc-server.example.com`?

Reply:
Hrm. Not a bad idea!

Reply:
I don't think it makes much difference, both `irc` or `server` (or even `ir-server`) look clear to me. Go with the one you like guys :)